### PR TITLE
Use import/export syntax and use createElement from react-native-web

### PR DESCRIPTION
**Pull request by EyMaddis**
This aligns the project with other React-Native packages and allows to use the style behavior of React-Native-Web (`StyleSheet.create`).

### CODE REVIEW

**Comment by joan-saum** [comment]
Hey @EyMaddis , it looks like @bakerface appeared recently, so if you resolve the conflict, the PR could be merged :)

**Comment by EyMaddis** [comment]
@joan-saum @bakerface Is up to date :)

File: index.js [hunk 1]
```diff
@@ -20,53 +20,100 @@
...
+export const Marker = createElement("Marker", "marker");
+Svg.Marker = Marker;
+
+export const Mask = createElement("Mask", "mask");
+Svg.Mask = Mask;
+
+export const Path = createElement("Path", "path");
+Svg.Path = Path;
+
+export const Path = createElement("Pattern", "pattern");
```
Comment (i2xzy) [inline index.js]: Could you add a fix for #7 by changing 
```js
export const Path = createElement("Pattern", "pattern");
Svg.Path = Path;
```
to:
```js
export const Pattern = createElement("Pattern", "pattern");
Svg.Pattern = Pattern;
```

**Comment by bakerface** [comment]
@EyMaddis thank you for the pull request, this has been released to v1.0.3

**Comment by i2xzy** [comment]
I'm getting this error:
```
ERROR in ./node_modules/react-native-svg-web/index.js 103:13
Module parse failed: Identifier 'Svg' has already been declared (103:13)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
| Svg.Stop = Stop;
| 
> export const Svg = createElement("Svg", "svg");
| Svg.Svg = Svg;
| 
```

Looks like the problem is this line was added:
```js
Svg.Svg = Svg; // stay consistent with the old require based api
```
But it didn't need to be as the original author already included Svg as a named export:
```js
Svg.Svg = createElement('Svg', 'svg');
```
which was changed to this in the pr:
```js
export const Svg = createElement("Svg", "svg");
Svg.Svg = Svg;
```

**Comment by EyMaddis** [comment]
Sorry to both of you.
Should be fixed in https://github.com/bakerface/react-native-svg-web/pull/9
Since I don't actually use the library right now I did not properly test... Shame on me.

**Comment by i2xzy** [comment]
ok I'm getting a different error now:
```
WARNING in ./node_modules/react-native-svg-web/index.js 29:11-26
"export 'createElement' (imported as 'RNcreateElement') was not found in 'react-native-web'
```

**Comment by EyMaddis** [comment]
This depends on the react-native-web version. In the latest the import was renamed to unstable_createElement..
Here is the way react-native-svg solves this:
https://github.com/react-native-community/react-native-svg/blob/develop/src/ReactNativeSVG.web.ts#L14

**Comment by i2xzy** [comment]
Probably best to do it that way to stay backwards compatible

**Pull request merged**